docs/reference/snapshot-api.asciidoc

```diff
@@ -0,0 +1,184 @@
+[[snapshot-restore-api]]
+=== Snapshot Restore API
+++++
+<titleabbrev>Snapshot Restore API</titleabbrev>
+++++
+////
+Restores a snapshot into a running cluster.
+////
+
+Restores a snapshot into a running cluster.
+
+[source,js]
+--------------------------------------------------
+POST /_snapshot/my_backup/snapshot_1/_restore
+--------------------------------------------------
+// CONSOLE
+
+////
+***********************************************************
+////
+
+////
+Use the appropriate heading levels for your book.
+Add anchors for each section.
+FYI: The section titles use attributes in case those terms change.
+////
+
+[[sample-api-request]]
```

Comment:
This looks like the placeholder ID from the template. This needs to be unique to the API. Here's a suggestion:

```suggestion
[[snapshot-restore-api-request]]
```